source/developers/dotnet/quickstart.md

```diff
@@ -176,9 +176,9 @@ dotnet test ./Minio.Tests/Minio.Tests.csproj
 * [SetPolicy.cs](https://github.com/minio/minio-dotnet/blob/master/Minio.Examples/Cases/SetBucketPolicy.cs)
 
 #### Bucket notification Operations
-* [GetBucketNotification.cs](./Minio.Examples/Cases/GetBucketNotification.cs)
-* [SetBucketNotification.cs](./Minio.Examples/Cases/SetBucketNotification.cs)
-* [RemoveAllBucketNotifications.cs](./Minio.Examples/Cases/RemoveAllBucketNotifications.cs)
+* [GetBucketNotification.cs](/Minio.Examples/Cases/GetBucketNotification.cs)
```

Comment:
@djwfyi I think the problem is these need to be extlinks - like in line 176. If we replace with the full URL this might just work :tm: